### PR TITLE
docs: update readme and remove requirements

### PR DESCRIPTION
**Pull request by skoh7645**
### Checklist:

- [x] Read the [Code of Conduct](https://github.com/appsody/website/blob/master/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md).

- [x] Followed the [commit message guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).

- [x] Stack adheres to [Appsody stack structure](https://github.com/appsody/website/blob/master/content/docs/stacks/stacks-overview.md#stack-structure).

Updated `README.md` and removed `TECHNICAL_REQUIREMENTS.md`

### Related Issues:

Related to: https://github.com/appsody/website/issues/199
This should only be merged once https://github.com/appsody/website/pull/483 goes in.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,26 +5,26 @@
...
+- `stable/`: Stable stacks are production-ready.
 
-- `incubator/:` The stacks in the incubator folder are actively being worked on to satisfy the stable criteria.
+- `incubator/`: Incubator stacks are actively being worked on to satisfy the stable criteria.
 
-- `experimental/:` Experimental stacks are not being actively been worked on and may not fulfill the requirements of an Appsody stack. These can be used for trying out specific capabilities or proof of concept work.
+- `experimental/`: Experimental stacks are not being actively been worked on and may not fulfill the requirements of an Appsody stack. These can be used for trying out specific capabilities or proof of concept work.
 
-Click here to find out more about [Appsody stacks](https://github.com/appsody/website/blob/master/content/docs/stacks/stacks-overview.md).
+Click here to find out more about [Appsody stacks](https://appsody.dev/docs/stacks/stacks-overview#stack-stability-levels).
```
Comment (neeraj-laad) [inline README.md]: perhaps the link test could include stability levels to make it clearer it is diff from the link above.

File: README.md [hunk 2]
```diff
@@ -5,26 +5,26 @@
...
 
 ## Creating a Stack
 We are actively working to create new stacks so that more people can adopt Appsody. If you find that none of the existing stacks meet your needs please reach out to us on the [Appsody Slack](https://appsody-slack.eu-gb.mybluemix.net/) or create a new GitHub issue to track the discussion.
```
Comment (neeraj-laad) [inline README.md]: We could add a link here to developing stack document?

Comment (skoh7645) [reply README.md]: I've changed the link from "creating a stack" (which has been removed) to the "developing a stack" doc

**Comment by neeraj-laad** [comment]
Looks good to me. Might be worth a quick review from @helenmasters

File: README.md [hunk 3]
```diff
@@ -5,31 +5,31 @@
 
 #### Compose a cloud native masterpiece.
 
-Infused with cloud native capabilities from the moment you start, Appsody provides everything you need to iteratively develop applications, ready for deployment to Kubernetes environments. Teams are empowered with sharable technology stacks, configurable and controllable through a central hub.
+Infused with cloud native capabilities from the moment you start, Appsody provides everything you need to iteratively develop applications, ready for deployment to Kubernetes environments. Teams are empowered with sharable technology stacks, configurable and controllable through a central hub. To find out more about Appsody check out [appsody.dev](https://appsody.dev).
```
Comment (helenmasters) [inline README.md]: Can you put in a comma after i.e. "To find out more about Appsody, check out ..."

File: README.md [hunk 4]
```diff
@@ -5,31 +5,31 @@
...
+Click here to find out more about [Appsody stack stability levels](https://appsody.dev/docs/stacks/stacks-overview#stack-stability-levels).
 
 ## Creating a Stack
 We are actively working to create new stacks so that more people can adopt Appsody. If you find that none of the existing stacks meet your needs please reach out to us on the [Appsody Slack](https://appsody-slack.eu-gb.mybluemix.net/) or create a new GitHub issue to track the discussion.
 
-We always welcome any contributions. If you wanted to create your own stack for a framework or language that we do not currently support, please review the [contributing guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md) and follow the steps outlined in [creating a stack](https://github.com/appsody/website/blob/master/content/docs/stacks/create.md).
+We always welcome any contributions. If you wanted to create your own stack for a framework or language that we do not currently support, please review the [contributing guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md) and follow the steps outlined in [developing a stack](https://appsody.dev/docs/stacks/develop).
```
Comment (helenmasters) [inline README.md]: How about streamlining it to: "To create your own stack for a framework or language that is not currently supported, review the [contributing guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md) and follow the steps that are outlined in [developing a stack](https://appsody.dev/docs/stacks/develop)."

File: README.md [hunk 5]
```diff
@@ -5,31 +5,31 @@
...
+- `incubator/`: Incubator stacks are actively being worked on to satisfy the stable criteria.
 
-- `experimental/:` Experimental stacks are not being actively been worked on and may not fulfill the requirements of an Appsody stack. These can be used for trying out specific capabilities or proof of concept work.
+- `experimental/`: Experimental stacks are not being actively been worked on and may not fulfill the requirements of an Appsody stack. These can be used for trying out specific capabilities or proof of concept work.
 
-Click here to find out more about [Appsody stacks](https://github.com/appsody/website/blob/master/content/docs/stacks/stacks-overview.md).
+Click here to find out more about [Appsody stack stability levels](https://appsody.dev/docs/stacks/stacks-overview#stack-stability-levels).
 
 ## Creating a Stack
 We are actively working to create new stacks so that more people can adopt Appsody. If you find that none of the existing stacks meet your needs please reach out to us on the [Appsody Slack](https://appsody-slack.eu-gb.mybluemix.net/) or create a new GitHub issue to track the discussion.
```
Comment (helenmasters) [inline README.md]: Can you put a comma in after the word needs please?

File: README.md [hunk 6]
```diff
@@ -5,31 +5,31 @@
...
 
-We always welcome any contributions. If you wanted to create your own stack for a framework or language that we do not currently support, please review the [contributing guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md) and follow the steps outlined in [creating a stack](https://github.com/appsody/website/blob/master/content/docs/stacks/create.md).
+We always welcome any contributions. If you wanted to create your own stack for a framework or language that we do not currently support, please review the [contributing guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md) and follow the steps outlined in [developing a stack](https://appsody.dev/docs/stacks/develop).
 
 ## Need help?
 If you have a question that you can't find an answer to, we would also like to hear about that too. You can reach out to the community for assistance on [Slack](https://appsody-slack.eu-gb.mybluemix.net/).
```
Comment (helenmasters) [inline README.md]: As you're editing in here please remove the word "too".

File: README.md [hunk 7]
```diff
@@ -5,31 +5,31 @@
...
+Infused with cloud native capabilities from the moment you start, Appsody provides everything you need to iteratively develop applications, ready for deployment to Kubernetes environments. Teams are empowered with sharable technology stacks, configurable and controllable through a central hub. To find out more about Appsody check out [appsody.dev](https://appsody.dev).
 
 # Stacks
 
 This repository holds the default set of stacks available from Appsody. Each stack is pre-configured with popular open source technologies to enable rapid development of quality microservice-based applications. Stacks include a base container image and project templates which act as a starting point for your application development.
 
 Stacks allow for rapid development whilst giving the stack developer the ability to control the overall applications that are created from it.
 
-To find out more about Appsody check out [appsody.dev](https://appsody.dev).
+Click here to find out more about [Appsody stacks](https://appsody.dev/docs/stacks/stacks-overview).
```
Comment (helenmasters) [inline README.md]: For more information, see [Appsody stacks](https://appsody.dev/docs/stacks/stacks-overview).

File: README.md [hunk 8]
```diff
@@ -5,31 +5,31 @@
...
 Stacks are categorized as either stable, incubator or experimental depending on the content of the stack.
 
-- `stable/`: Stable stacks meet this set of [technical requirements](https://github.com/appsody/stacks/blob/master/TECHNICAL_REQUIREMENTS.md).
+- `stable/`: Stable stacks are production-ready.
 
-- `incubator/:` The stacks in the incubator folder are actively being worked on to satisfy the stable criteria.
+- `incubator/`: Incubator stacks are actively being worked on to satisfy the stable criteria.
 
-- `experimental/:` Experimental stacks are not being actively been worked on and may not fulfill the requirements of an Appsody stack. These can be used for trying out specific capabilities or proof of concept work.
+- `experimental/`: Experimental stacks are not being actively been worked on and may not fulfill the requirements of an Appsody stack. These can be used for trying out specific capabilities or proof of concept work.
```
Comment (helenmasters) [inline README.md]: This description is doesn't make sense "being actively been worked on" - why can't we match the descriptions we've just edited on the overview page for these categories? e.g. "These stacks are not production-ready and are considered as proof of concept. They might be unstable, and subject to breaking changes." for experimental etc etc?

Comment (skoh7645) [reply README.md]: I'll change these to use the same descriptions as on the website :)

**Pull request merged**